cmake/SSGCommon.cmake

```diff
@@ -1257,6 +1257,57 @@ macro(ssg_build_html_nistrefs_table PRODUCT PROFILE)
         DESTINATION "${SSG_TABLE_INSTALL_DIR}")
 endmacro()
 
+
+macro(ssg_build_html_ref_table BASENAME PRODUCT REFERENCE REF_FORMAT)
+    add_custom_command(
+        OUTPUT "${CMAKE_BINARY_DIR}/tables/${BASENAME}.html"
+        COMMAND "${CMAKE_COMMAND}" -E make_directory "${CMAKE_BINARY_DIR}/tables"
+        COMMAND env "PYTHONPATH=$ENV{PYTHONPATH}" "${PYTHON_EXECUTABLE}" "${CMAKE_SOURCE_DIR}/utils/gen_reference_table.py" --build-dir "${CMAKE_BINARY_DIR}" --output "${CMAKE_BINARY_DIR}/tables/${BASENAME}.html" --ref-format "${REF_FORMAT}" "${PRODUCT}" "${REFERENCE}"
```

Comment:
I would execute the script only once and generate them in a single step.

Reply:
That would mean introducing a `generate_references_tables ... <output filename template> <ref1> <ref2> <ref3> ...` script - I will check that out.

Reply:
That should be possible - you can get inspired in the PR https://github.com/ComplianceAsCode/content/blob/ef14ad67b67e58272889d410391f196a5f449842/cmake/SSGCommon.cmake#L196